dev/DynamicDependency/M.AM.Converters.h

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
is htere a reason you didn't spell out "Microsoft.ApplicationModel" in the file name?

Reply:
Long highly redundant filenames get awkward to deal with (Visual Studio and VSCode source panes, etc).

Reply:
I find this pattern confusing and unconventional, and would prefer the full name for searching by filename to work as expected.